app/mailers/portfolio_evidence_mailer.rb

```diff
@@ -84,4 +84,19 @@ def portfolio_failed(project)
     subject = "#{project.unit.name}: Portfolio failed to compile"
     mail(to: email_with_name, from: convenor_email, subject: subject)
   end
+
+  def portfolio_auto_ready(project)
+    return nil if project.nil?
+
+    add_general
+
+    @student = project.student
+    @project = project
+    @convenor = project.unit.convenors.first.user
+
```

Comment:
@macite should these mailers also say which unit it has been generated for? I was unsure whether to add it since the existing mailers didn't use it.

Reply:
@C-Blenco yes that would be a good addition.

Reply:
@macite no worries, did you want me to update the other mailers to also include the unit code?